src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -112,10 +112,8 @@ export const CollectionFeaturedArtistsContainer = createFragmentContainer(Featur
   collection: graphql`
     fragment FeaturedArtists_collection on MarketingCollection
       @argumentDefinitions(screenWidth: { type: "Int", defaultValue: 500 }) {
-      # TODO: size:9 is not actually limiting to 9 items. We need to figure out
-      #  why the back-end is not respecting that argument.
-      artworksConnection(aggregations: [MERCHANDISABLE_ARTISTS], size: 9, sort: "-decayed_merch") {
-        merchandisableArtists {
+      artworksConnection(aggregations: [MERCHANDISABLE_ARTISTS], size: 0, sort: "-decayed_merch") {
+        merchandisableArtists(size: 9) {
```

Comment:
I used 9, since that what was already here.

Also note that since we only want the artists in this fragment, it's ok to provide size: 0 to the artworksConnection, since we are not actually rendering any artworks.

(I concede this is weird, but it has to do with the underlying Elasticsearch functionality, where you can request 0 matching hits, but still trigger aggregations over the would-be result set.)

Reply:
So the takeaway here is we were getting more than 9 artists in the request because we weren't restricting the size to 9 artists, but rather 9 artworks...and we can actually request 9 artists in the aggregated query and 0 artworks.